src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,6 +8,7 @@ class Program
     {
         static int Main(string[] args)
         {
+            Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
             try
```

Comment:
@Alexx999, I think we're almost on the same page... my point was that just as the functionality isn't locale-dependent, the test shouldn't be locale-dependent either, and thus we shouldn't be hardcoding to a particular locale.  Regardless of locale, doing comparisons against strings produced by the runtime also makes the test susceptible to any changes that get made to the strings produced by the runtime, e.g. if it were to change the spacing in the exception ToString message, the test would start to fail, even though the functionality being tested is still working correctly.  As such, we shouldn't be depending on string comparisons against a particular string produced by code other than in the test, and thus we shouldn't need to lock to a particular locale.  I'm fine with a solution that just outputs a particular string to stderr rather than outputting the exception text.